release-notes/opensearch-index-management-dashboards-plugin.release-notes-2.3.0.0.md

```diff
@@ -0,0 +1,17 @@
+## Version 2.3.0.0 2022-09-09
+
+Compatible with OpenSearch 2.3.0
+
+### Enhancement 
```

Comment:
Hi @phestecAMZN this should have been `Enhancements` instead of `Enhancement`
This might cause automation to skip this section in future. Not sure if these were manually written or automated but need to fix this.
Thanks!

Reply:
cc: @opensearch-project/index-management